### PR TITLE
Release v1.37.0

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix

/kind release

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Work on #3556 

**Special notes for your reviewer**:

### CODE REVIEW

**Review by gongmax — APPROVED** [review]

File: site/content/en/blog/releases/1.37.0.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+This is the 1.37.0 release of Agones.
+
+This release sees the first milestone of a over-a-year series of work on the functionality of arbitrary Counters and Lists attached to each GameServer!
+
+We’ve also upgraded the following features: CountsAndLists to alpha, FleetAllocationOverflow to Beta, and PodHostName, ResetMetricsOnDelete, and SplitControllerAndExtensions to Stable - with defaults below.
+
+- CountsAndLists: This feature flag provides functionality for a configurable set of Counters or Lists be attached to GameServers with attached allocation and autoscaling functionality to provide a flexible implementation that can handle a wide range of use cases such as player tracking, high density game server room tracking, and could even be used for game specific game server allocation prioritization.
+
+- Documentation is a little sparse in this release, but will grow! We’re focusing on the reference material first and will then expand into more in-depth guides and integration patterns.
```
Comment (roberthbailey) [inline site/content/en/blog/releases/1.37.0.md:15]: This and the next paragraph shouldn't have new bullets in the bulleted list. Let me see if I can tinker with the markdown to make it look a bit better.

Comment (Kalaiselvi84) [reply site/content/en/blog/releases/1.37.0.md:15]: Does it look good?

https://screenshot.googleplex.com/5hYyMBcWTg7ryJq

Comment (roberthbailey) [reply site/content/en/blog/releases/1.37.0.md:15]: Yes - that's what I was going for :)

File: site/content/en/blog/releases/1.37.0.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+- Note: Counters and Lists will eventually replace the Alpha functionality of Player Tracking, which will subsequently be removed from Agones. If you are currently using this Alpha feature, we would love for you to test (and ideally migrate to!) this new functionality to ensure it will meet all your needs.
```
Comment (roberthbailey) [inline site/content/en/blog/releases/1.37.0.md:17]: Can you make "note" here be bold, e.g. `**Note**:`

File: site/content/en/blog/releases/1.37.0.md (hunk 1 above)
Comment (Kalaiselvi84) [reply site/content/en/blog/releases/1.37.0.md:15]: Quick question:

Do we need to delete the `data-proofer-ignore` attribute from the files 
`site/content/en/docs/Guides/Client SDKs/rust.md` and `site/content/en/docs/Third Party Content/examples.md`?

https://screenshot.googleplex.com/9Wx4Pt9mCo6AmtU

https://screenshot.googleplex.com/97QN9ZLDoyQgLav

File: site/content/en/blog/releases/1.37.0.md (hunk 2 above)
Comment (Kalaiselvi84) [reply site/content/en/blog/releases/1.37.0.md]: This change is in place.

**Review by roberthbailey — APPROVED** [review]

**Comment by roberthbailey** [comment]
Test failure: 
```
Step #1 - "e2e-feature-gates": VERBOSE: I1219 21:15:48.706046    9317 request.go:696] Waited for 1.910837524s due to client-side throttling, not priority and fairness, request: POST:https://34.74.54.7/apis/allocation.agones.dev/v1/namespaces/1703020507/gameserverallocations
Step #1 - "e2e-feature-gates": VERBOSE:     gameserverallocation_test.go:1397: could not completed gsa1 allocation : an error on the server ("error updating allocated gameserver: Operation cannot be fulfilled on gameservers.agones.dev \"preferred-9z6d7-4v752-gvlvf\": the object has been modified; please apply your changes to the latest version and try again") has prevented the request from succeeding (post gameserverallocations.allocation.agones.dev)
Step #1 - "e2e-feature-gates": VERBOSE: time="2023-12-19 21:15:55.323" level=info msg="Finished Allocation."
Step #1 - "e2e-feature-gates": VERBOSE:     gameserverallocation_test.go:1417: 
Step #1 - "e2e-feature-gates": VERBOSE:         	Error Trace:	/go/src/agones.dev/agones/test/e2e/gameserverallocation_test.go:1417
Step #1 - "e2e-feature-gates": VERBOSE:         	Error:      	Not equal: 
Step #1 - "e2e-feature-gates": VERBOSE:         	            	expected: 100
Step #1 - "e2e-feature-gates": VERBOSE:         	            	actual  : 99
Step #1 - "e2e-feature-gates": VERBOSE:         	Test:       	TestGameServerAllocationDuringMultipleAllocationClients
Step #1 - "e2e-feature-gates": VERBOSE: --- FAIL: TestGameServerAllocationDuringMultipleAllocationClients (47.23s)
Step #1 - "e2e-feature-gates": VERBOSE: FAIL
Step #1 - "e2e-feature-gates": VERBOSE: time="2023-12-19 21:15:55.929" level=info msg="Namespace 1703020507 is deleted"
Step #1 - "e2e-feature-gates": VERBOSE: FAIL	agones.dev/agones/test/e2e	49.261s
```

**Pull request merged**